src/components/Form.js

```diff
@@ -1,34 +1,67 @@
-import { useState } from 'react'
+import { useState, useEffect, createRef } from 'react'
 import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
 import Button from './Button'
 import Input from './Input'
+import { CrossIcon } from '../lib/Icons'
 
 function Form({ userPlace, setUserPlace, status }) {
     const [userInput, setUserInput] = useState('')
+    const [validation, setValidation] = useState(true)
+
+    const textInput = createRef()
```

Comment:
`createRef` is the old way to set references (like in class components).
In functional components, please use [`useRef`](https://reactjs.org/docs/hooks-reference.html#useref).